.github/workflows/py36.yml

```diff
@@ -1,10 +1,6 @@
 name: unittests py36
 
-on:
-  push:
-    branches: [main]
-  pull_request:
-    branches: [main]
+on: [pull_request, push]
```

Comment:
I think with this notation you don't specify to which branch the actions will trigger. meaning they will trigger on any branch to branch merge/PR. I would use:

```yaml
on:
  push:
    branches: [master]
  pull_request:
    branches: [master]
 ```

unless you really know what you are doing, hence I rest my case :relaxed: